.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-- Close #https://github.com/nimblehq/ios-templates/issues/
+- Close #
```

Comment:
Optional: As this is a template, you might want to consider adding a description with brackets for better clarity 🙏
```suggestion
- Close #{issue_no}
```

Reply:
I think the purpose of this part of the template is so that it's easy to autofill. Adding the description will make it one more step to write the PR.
When user uses this enough time there wouldn't be reading the description anymore 🙏 .